libfreerdp/core/gateway/rts.c

```diff
@@ -887,7 +887,7 @@ int rts_recv_flow_control_ack_pdu(rdpRpc* rpc, BYTE* buffer, UINT32 length)
 	BYTE ChannelCookie[16];
 
 	offset = 24;
-	offset += rts_flow_control_ack_command_read(rpc, &buffer[offset], length - offset,
```

Comment:
Commenting out rather than removing is definitely much better for me. However, you said it well: clang warnings do not understand readability concerns. Are we really in favor of making the code less readable for the sake of not having clang warnings?